toc/working-groups/app-runtime-platform.md

```diff
@@ -111,6 +113,7 @@ Components from the Diego, Garden, HAproxy, Logging and Metrics, Networking, Win
   * https://github.com/cloudfoundry/eventhub
   * https://github.com/cloudfoundry/executor
   * https://github.com/cloudfoundry/fileserver
+  * https://github.com/cloudfoundry/grace
```

Comment:
Not sure if this a good example app for testing CF:
- last commit 2y
- no CF manifest
- dockerfile based on busybox:ubuntu-14.04

But I would not veto if the WG insists it is needed.

Reply:
It may not be a good example app, but it's still being used by diego's tests. Additionally, [https://cloudfoundry.slack.com/archives/C026XJGNC2U/p1637084567022300?thread_ts=1637083979.022000&cid=C026XJGNC2U](this conversation) made it seem like we would want to list it anyway.

Maybe ultimately we replace it in diego's tests, and delete the repo?